lib/react-step-wizard.jsx

```diff
@@ -37,6 +37,38 @@ var StepWizard = React.createClass({
     this.validateChildren();
   },
 
+  componentDidMount: function() {
+    if(window.addEventListener) {
+      window.addEventListener('popstate', this.navigateBack);
+    } else if (window.attachEvent) {
+      window.attachEvent('onpopstate', this.navigateBack);
+    } else {
+      window.onpopstate = this.navigateBack;
+    }
+
+    window.history.replaceState({currentStepIndex: 0}, 'Step 0');
+  },
+
+  componentWillUnmount: function() {
```

Comment:
componentWillUnmount will not fire for our application, should be attached to window.onbeforeunload